xml/System.ComponentModel.Composition/AdaptationConstants.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains static metadata keys used by the composition system.</summary>
     <remarks>To be added.</remarks>
```

Comment:
Should the remarks be deleted?

Reply:
no seems like we keep them since they don't appear in the doc builds, and because they make it easier to identify sections with no content.